client/public/index.html

```diff
@@ -25,6 +25,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>BobaTalks Internship Tracker</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
Can we bring fonts in a the theme level? Check out https://mui.com/material-ui/customization/theming/

We should be able to bring the font in (download and save as a font asset if we need to) and assign fonts with the MUI theme so any typography elements will use this

Reply:
Just remember we do have the theme, we may need to just download the font files and save them in our assets

Reply:
can I ask why we need to download instead of importing the fonts? the font is already added to the theme but doesn't show because it wasnt imported (that's why i've imported it from google fonts). I've seen many occurrences where google fonts are imported in css files or added to index.html so I am just curious as to why you suggested downloading it.

Reply:
Try https://www.npmjs.com/package/@fontsource/poppins